src/monitoring/mod.rs

```diff
@@ -104,6 +105,20 @@ pub fn increment_btc_blocks_received_counter() {
     prometheus::BTC_BLOCKS_RECEIVED_COUNTER.inc();
 }
 
+#[allow(unused_variables)]
+pub fn set_last_execution_cost_observed(execution_cost: &ExecutionCost) {
+    #[cfg(feature = "monitoring_prom")]
+    prometheus::LAST_EXECUTION_READ_COUNT.set(execution_cost.read_count.try_into().unwrap());
```

Comment:
We can try to make these percentages.

But, is this the right structure to get the data into prometheus?

Reply:
Yeah, this works.

Instead of panicking on failing to convert from the costs to `i64`, can you just saturate to `i64::max`?

Reply:
what is the command to saturate?

Reply:
what would be a good way to get the active ExecutionCost limits in this `append_block` context @kantai ?

Reply:
```rust
execution_cost.read_count.try_into().unwrap_or(i64::MAX)
```

Reply:
> what would be a good way to get the active ExecutionCost limits in this append_block context @kantai ?

`append_block` creates a `clarity_tx` variable for execution of the block. That `ClarityTx` type has a `block_limit()` instance method -- it returns an `Option`, but it should _always_ be `Some`, as long as there is no transaction currently being executed.

Reply:
should be as a percentage right now